### PR TITLE
Fixed Posts

### PR DESCRIPTION
**Pull request by jchomali**
Fixed posts not displaying and deleted unnecessary code line when looking for the timeCreatedString

### CODE REVIEW

**Comment by jchomali** [comment]
I didn't updated libHN in the version 3.1.1, so is not possible to merge this fix.

**Comment by amitburst** [comment]
Posts are broken for me too!

**Comment by jchomali** [comment]
@amitburst take a look at this commit https://github.com/jchomali/libHN/commit/55b106f0a11b2c576ddeb097cd1e9f3862c4ea73 it will fix your problem

**Comment by amitburst** [comment]
Thanks, @jchomali. However, I'm maintaining an open source repository that uses this library, so I don't want to deal with the complexity of using my own fork. I will wait until this fix gets merged in. :)

**Comment by bennyguitar** [comment]
I appreciate the pull request, but I decided that I'm tired of fixing small incremental markup changes so now I have an `hn.json` configuration file that is loaded into the bundle that determines how to parse posts and comments. `HNManager` also now downloads a new configuration file (from this github repo) whenever it initializes itself and determines if the configuration needs to change. This means that whenever your app uses this library and HN changes their markup, this file just needs a pull request to handle the new markup and your apps should work automatically.

TL;DR: no more submitting new apps to handle markup changes with this library.

Check the README tomorrow and I'll have a whole section on how to edit/format `hn.json` and what this means for your apps that use `4.0.0` or higher of this library. You can use the new version starting right now. :)

**Comment by jchomali** [comment]
@bennyguitar Thank you for that update!! 👏👏

**Comment by auto234875** [comment]
Thanks for the hard work Ben. I'm getting sick of the way HN is treating us.
